README.md

```diff
@@ -1,13 +1,40 @@
-# OpenZeppelin Runtime Template for Substrate
+# OpenZeppelin Runtime Templates for Substrate
 
 Substrate has a steep learning curve. In order to make it easier for newcomers to get started, OpenZeppelin is providing runtime templates that can be used to quickly bootstrap a Substrate project.
 
 Along with the templates, OpenZeppelin also provides a documentation website that explains the templates and the niche details.
 
+## About this repository and templates
+
+This is a monorepo, listing all of our templates. Each template has its own sub-directory, and completely independent from other templates.
+
+For example: if you want to use our `generic runtime template`, you can just copy that subdirectory, and it will work on its own. You don't need anything from other templates nor from the root directory of this repository for your selected template to work.
+
+---
+
```

Comment:
Let's take the oportunity and add a warning so that people don't use `master`. Something like this should be enough:
```suggestion
> [!WARNING]
> Avoid using `main` for production deployments and use release branches instead. `main` is a development branch that should be avoided in favor of tagged releases. The release process involves security measures that the `main` branch does not guarantee.
```